apps/expo/app.config.ts

```diff
@@ -32,7 +32,7 @@ const defineConfig = (): ExpoConfig => ({
       // projectId: "your-project-id",
     },
   },
-  plugins: ["./expo-plugins/with-modify-gradle.js"],
+  plugins: ["expo-router", "./expo-plugins/with-modify-gradle.js"],
```

Comment:
what does this do?

i don't see this mentioned in any docs + we already have it in the babel.config

Reply:
Well that is interesting! I thought I followed [the guide](https://blog.expo.dev/introducing-expo-router-v2-3850fd5c3ca1) exactly, but now I can't find it. I guess I misread it. I just ran it without that line and it was fine. So I'll push a change that drops it.